presto-product-tests/src/main/java/io/prestosql/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -186,4 +189,26 @@ public void testAccessControlDropView()
         assertThat(() -> aliceExecutor.executeQuery(format("SELECT * FROM %s", viewName)))
                 .failsWithMessage("does not exist");
     }
+
+    @Test(groups = {AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
+    public void testAccessControlSelectWithCaseSensitiveUserName()
+    {
+        assertThat(() -> caseSensitiveUserNameExecutor.executeQuery(format("SELECT * FROM %s", tableName)))
+                .failsWithMessage(format("Access Denied: Cannot select from table default.%s", tableName));
+
+        onHive().executeQuery("SET ROLE admin");
+
+        // make sure that case matters
+        onHive().executeQuery(format("GRANT SELECT ON TABLE %s TO USER casesensitiveusername", tableName));
```

Comment:
Is there such user as `casesensitiveusername` in Hive?

Reply:
What do you mean? Hive does not store users. It stores information about privileges and user comes from outside.

Reply:
True. Forgot that